modules/pages/client/components/Donate.component.js

```diff
@@ -0,0 +1,160 @@
+              )}
+            </p>
+            <br />
+
+            <h4>{t('Is my donation tax deductible?')}</h4>
+            <p>{t('Unfortunately not at the moment.')}</p>
+            <br />
+
+            <h4>{t('Which currencies do you accept?')}</h4>
+            <p>
+              {t(
+                'You can donate via PayPal, which currently accepts nearly two dozen currencies.',
+              )}
+            </p>
+            <br />
+
+            <h4>{t('Can I donate bitcoin?')}</h4>
+            <p>{t('Yes!')}</p>
+            <br />
+
+            <h4>
+              {t(
+                'I’m unable to give money at this time, are there other ways I can help?',
+              )}
+            </h4>
+            <p>
+              <Trans t={t} ns="pages">
+                Yes! <a href="/volunteering">Become a volunteer</a> and make a
+                difference.
+              </Trans>
+            </p>
+
+            <hr />
+
+            <p>
+              <a href="/donate/help">{t('Problems donating?')}</a>
+            </p>
+
+            <br />
+          </div>
+          <div className="col-xs-12 col-sm-6 col-md-5">
+            <p className="lead">
+              <em>
+                {t(
+                  "We're currently working on setting up ways to receive donations. Until that, this page gives you preliminary information about our funding model.",
+                )}
+              </em>
+            </p>
+            <hr />
+            <h2>{t('Ways to Give')}</h2>
+            <h3>{t('Credit/debit card')}</h3>
+            {t(
+              "We don't have a credit card transfer set up yet at this point.",
+            )}
+            <h3>{t('Bank transfer')}</h3>
+            {t("We don't have a bank account set up yet at this point.")}
+            <h3>Bitcoin</h3>
+            <Trans t={t} ns="pages">
+              <a href="/support">Contact us</a> if you want to donate through
+              Bitcoin.
+            </Trans>
+            <h3>PayPal</h3>
+            <Trans t={t} ns="pages">
+              <a href="/support">Contact us</a> if you want to donate through
+              PayPal.
+            </Trans>
+            <h3>{t('Referal programs')}</h3>
+            <h4>{t('Namecheap')}</h4>
+            <Trans t={t} ns="pages">
+              We use and recommend Namecheap for domains, partly because
+              they&apos;re a{' '}
+              <a href="https://www.namecheap.com/about/causes.aspx">
+                responsible company
+              </a>{' '}
+              and foremost they do a great job. Sign up via{' '}
+              <a href="https://www.namecheap.com/?aff=90636">this link</a> and
+              we&apos;ll get little bit in return.
+            </Trans>
+            <br />
+            <h4>DigitalOcean</h4>
+            <Trans t={t} ns="pages">
+              Our servers are hosted at reliable and easy to use DigitalOcean.
+              Sign up via{' '}
```

Comment:
I'll be interested to see how this turn out in the i18n json files... I also feel like there could be another solution, but not sure what it is:
- `&nbsp;` is maybe just as ugly
- disabling prettier/eslint rules for these `Trans` blocks
- maybe some CSS solution to not collapse white-space?

Reply:
For me, this is an HTML problem needing an HTML solution, cause we would need to insert an optional space or line break in case that an inline element follows the Trans block. I find that {' '} is the best I know since
- it breaks when necessary
- it creates a space when needed
- it only creates a space after the last character and not between the whole paragraph and the next element

I might miss out on something here but from what I understand, it appears the best solution to me that comes to my mind, given that it works out with the i18n json files.